boards/riscv/icev_wireless/icev_wireless.dts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2022 Friedt Professional Engineering Services, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3.dtsi>
+#include "icev_wireless-pinctrl.dtsi"
+
+/ {
+	model = "icev_wireless";
+	compatible = "espressif,esp32c3";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &usb_serial;
+		zephyr,shell-uart = &usb_serial;
+		zephyr,flash = &flash0;
+	};
+
+	aliases {
+		led0 = &led_green;
+		i2c-0 = &i2c0;
+		sw0 = &user_button1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_green: led_0 {
+			gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
```

Comment:
Is this C3 LED on the board? (guess the RGB goes to the FPGA). Maybe call it led-green in the node name as well...

Reply:
It's technically disjoint from the esp32 device tree. Were I to add a soft riscv core on the fpga, for example, then that would appear on a separate device tree (and would likely be in a separate file).

Reply:
Cool, found the schematics, makes sense.